manager/integration/tests/test_stress.py

```diff
@@ -160,6 +160,12 @@ def delete_random_snapshot(longhorn_api_client, volume_name):
         purge_random_snapshot(longhorn_api_client, volume_name, snapshot_name)
 
 
+def remove_datafile(pod_name):
+    file_path = '/tmp/data-' + pod_name + '.bin'
```

Comment:
Better to use constants for variables that referred more than once. The pod name prefix and suffix here.

Reply:
done